pkg/flowkit/config/json/config.go

```diff
@@ -33,6 +33,12 @@ type jsonConfig struct {
 	Accounts    jsonAccounts    `json:"accounts"`
 	Deployments jsonDeployments `json:"deployments"`
 }
+type defaultJsonConfig struct {
+	Contracts   jsonContracts   `json:"contracts"`
+	Networks    jsonNetworks    `json:"networks"`
+	Accounts    jsonAccounts    `json:"accounts"`
+	Deployments jsonDeployments `json:"deployments"`
+}
```

Comment:
Removed emulator from the original jsonconfig struct, because otherwise we will have emulator: null showing up in flow.json